tests/vmi_headless_test.go

```diff
@@ -41,7 +39,7 @@ var _ = Describe("[rfe_id:609][sig-compute]VMIheadless", decorators.SigCompute,
 		Context("with headless", func() {
 
 			It("[test_id:709][posneg:positive]should connect to console", func() {
-				vmi := libvmops.RunVMIAndExpectLaunch(libvmifact.NewAlpine(withAutoattachGraphicsDevice(false)), 30)
+				vmi := libvmops.RunVMIAndExpectLaunch(libvmifact.NewAlpine(libvmi.WithAutoattachGraphicsDevice(false)), 30)
```

Comment:
It is intentional to explicitly set the already optimized default false value?

Reply:
This being a headless test, it makes sense for it to ensure graphics are disabled.
If we ever changed the default back to true we'd want to make sure it stays false here.